shared/parser.go

```diff
@@ -350,7 +351,8 @@ func Parse(argList []string) ParserResult {
 		"-fopenmp":       {0, pr.compileLinkUnaryCallback},
 
 		"-Wl,-dead_strip": {0, pr.warningLinkUnaryCallback},
-		"-dead_strip":     {0, pr.warningLinkUnaryCallback}, //iam: tor does this. We lose the bitcode :-(
+		"-dead_strip":     {0, pr.warningLinkUnaryCallback},  //iam: tor does this. We lose the bitcode :-(
```

Comment:
Unnecessary reformatting?

Reply:
Done by `go fmt` but I'll try to figure out.

Reply:
Fixed in [0dff6f1](https://github.com/capablevms/gllvm/pull/1/commits/0dff6f1d1de40a9bde373988deabbd3216c9f49b), I *think*.